numpy/lib/financial.py

```diff
@@ -62,12 +62,10 @@ def fv(rate, nper, pmt, pv, when='end'):
 
     .. deprecated:: 1.18
 
-       `fv` is deprecated; see NEP 32::
-
-           https://numpy.org/neps/nep-0032-remove-financial-functions.html
-
-        Use the corresponding function in the numpy-financial library,
-        https://pypi.org/project/numpy-financial
+       `fv` is deprecated; for details, see `NEP 32
+       <https://numpy.org/neps/nep-0032-remove-financial-functions.html>`_.
```

Comment:
Hmm, this doesn't look very nice on console. I wonder if there is a nicer way to do this with a reference to the notes section the links as references?

Reply:
Add a tiny plugin to Sphinx to support `` :nep:`32` `` or similar?

Reply:
@charris, I just pushed a change in which I reference the NEP as "NEP 32 [1]_", and add the reference to the NEP to the "References" section. (I also tweaked the the other reference where appropriate for consistency.)

@eric-wieser, that might be nice, but then the docstring itself wouldn't have the URL of the NEP.